pyndemic/character.py

```diff
@@ -246,3 +246,18 @@ def standard_move(self, location, destination):
 
             return True
         return False
+
+    def check_action_card(self, card_name, *args):
+        card = self.get_card(card_name)
+        return card.check_playable(*args)
+
+    def play_action_card(self, card_name, *args):
+        if self.check_action_card(card_name, *args):
+            logging.info(
+                f'{self}: Playing {card_name}.')
```

Comment:
I would suggest to make a signal emitting here. This message seems to be a useful information for users, and `.emit_signal` includes logging by default.
```suggestion
        if self.check_action_card(card_name, *args):
            self.emit_signal(
                f'{self}: Playing {card_name}.')
```